CMakeLists.txt

```diff
@@ -3,6 +3,8 @@
 #on the platform and configuration it is set to build in.
 project(tinyobjloader)
 cmake_minimum_required(VERSION 2.8.6)
```

Comment:
Did you test this patch on CMake 2.8.6? I suspect this needs to be bumped. (I think 2.8.12 is sufficient.)